exploration/error-handling.md

```diff
@@ -0,0 +1,168 @@
+# Error Handling
+
+Status: **Proposed**
```

Comment:
I think this is good as is, but landing this as "proposed" would be fine as well.

```suggestion
Status: **Accepted**
```